### ISSUE TITLE
Migrate MQTT to Integrations

### ISSUE THREAD

**Issue by ubergesundheit** [sensebox/openSenseMap-API#71]
@noerw  Please write a migration on database to make sure mqtt works after the development branch is rolled out in production.

**Comment by noerw** [comment]
be careful though: run-migrations.sh will run **all** migrations

**Comment by ubergesundheit** [comment]
Yeah thanks, I have to think about an upgrade strategy when the frontend catches up to the implementations in the api (user stuff). I'll run all migrations then in one session
